web/src/pages/settings/Settings.tsx

```diff
@@ -1,44 +1,63 @@
 import React from "react";
 import Box from "@mui/material/Box";
 import Card from "@mui/material/Card";
+import Layout from "../../components/layout/Layout";
 
 function Settings() {
-	return (
-		<Box style={{ width: "100vw", height: "95vh" }}>
-			<h1 style={{ marginLeft: "5%" }}>Settings</h1>
-			<Box style={{ height: "80%" }}>
-				<SettingsGrid />
-			</Box>
-		</Box>
-	);
+  return (
+    <Layout>
+      <Box style={{ width: "100vw", height: "95vh" }}>
```

Comment:
C: Reduce `height` to `85vh` to remove scrollbar on page. Would ideally use percentages, but then the grid is not rendered, suspected 0 height.

Reply:
Commit: b0a39710a9dd49a1ffbe3b3c06c12cb371898c25